File: patterns/javascript-patterns/pattern8.js
// Pattern 8: ...

//     *
//    ***
//   *****
//  *******
// *********

const p_height = 5;
let min_stars = 1;
let p_space = p_height - 1;

for (let i = 0; i < p_height; i++) {
    let spaces = '';
    for (let j = p_space; j > i; j--) {
        spaces += ' ';
    }

    let stars = '';
    for (let k = 0; k < min_stars; k++) {
        stars += '*';
    }
    min_stars += 2;

    console.log(spaces + stars);
}

File: patterns/python-patterns/pattern8.py
# Pattern 8: ...

#     *
#    ***
#   *****
#  *******
# *********

p_height = 5
min_stars = 1

for i in range(p_height):
    for j in range(p_height - 1, i, -1):
        print(" ", end="")
    for k in range(min_stars):
        print("*", end="")
    min_stars += 2
    print()
